src-tauri/src/globalmincut.rs

```diff
@@ -0,0 +1,163 @@
+#![allow(dead_code)]
```

Comment:
Why do we need this directive?

Reply:
I was getting warnings (function x is never being used) and it bothered me lol

Reply:
Oh okay fair enough, let's keep an eye on that then into the future and make sure to remove them once they're not needed